doc/0.113/categories-v1.html

```diff
@@ -7,7 +7,11 @@
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   
+<<<<<<< HEAD:doc/next/categories-v1.html
```

Comment:
ugh.... merge gone wrong, or forgot to re-generate

Reply:
no clue how this can even happen, sphinx is a mystery to me

Reply:
I think with this PR https://github.com/ecg-icas/icas/pull/30 I removed many old generated versions. I'm going to redo that on this branch.